src/stapi_fastapi/models/order.py

```diff
@@ -115,3 +109,22 @@ def __len__(self) -> int:
     def __getitem__(self, index: int) -> Order:
         """get feature at a given index"""
         return self.features[index]
+
+
+class OrderPayload(BaseModel, Generic[ORP]):
```

Comment:
This was renamed from OrderRequest to OrderPayload so it's not semantically confused with a starlette/fastapi Request